diesel/src/pg/expression/expression_methods.rs

```diff
@@ -1779,7 +1790,7 @@ pub trait PgRangeExpressionMethods: Expression + Sized {
 impl<T> PgRangeExpressionMethods for T
 where
     T: Expression,
-    T::SqlType: RangeOrNullableRange,
+    T::SqlType: MultirangeOrRangeMaybeNullable,
```

Comment:
I can't test without flipping this, and this allows all the range operations to accept multirange without proper test. What do the reviewers think? Can I merge the PR like that and add tests in the future in other small PRs, or should I add a multirange test for all the range methods?

Reply:
The correct way to handle this would be to split up this trait into two:

* The `PgRangeExpressionMethods` contains methods that are only available for `Range` but not for `MultiRange`. That is then only implemented for `T::SqlType: RangeOrNullableRange`
* The `PgRangeAndMultirangeMethods` contains methods that are available for both
* 

(If there are methods that are only available for `MultiRange` we would just add a third trait for that.

Reply:
I did a fast check, and it looks like all the range operations are also multirange operations. If this is the case, do you still want to rename it to `PgRangeAndMultirangeMethods`?

I will add the tests for each of the supported methods in this PR

Reply:
In this case it should be fine to keep the old name.